src/policy/policy.cpp

```diff
@@ -73,7 +73,7 @@ bool IsStandard(const CScript& scriptPubKey, txnouttype& whichType)
                (!fAcceptDatacarrier || scriptPubKey.size() > nMaxDatacarrierBytes))
```

Comment:
Nit, add `{ }`.

Reply:
Planning to leave that since I'm not touching the line otherwise